sys/shell/commands/sc_sys.c

```diff
@@ -22,6 +22,9 @@
 
 #include "periph/pm.h"
 
+#define USB_H_USER_IS_RIOT_INTERNAL
+#include "usb_board_reset.h"
```

Comment:
```suggestion
#ifdef MODULE_USB_BOARD_RESET
#define USB_H_USER_IS_RIOT_INTERNAL
#include "usb_board_reset.h"
#endif
```

Reply:
The include is safe to use either way, but added.